app/models/account.rb

```diff
@@ -13,6 +13,8 @@ class Account < ApplicationRecord
   validates :name, uniqueness: {
     scope: [:user_id, :currency_id], case_sensitive: false }
 
+  validates :show_in_summary, inclusion: { in: [true, false] }
```

Comment:
I think it's not necessary to validate boolean fields. In the end, they are going to be stored as true/false anyway.
From the user perspective, we usually don't see validation errors for checkboxes.

I would suggest to remove this validation.